### DISCUSSION TITLE
New Flamerobin snapshot available! 0.9.3.5 with a few fixes and small features

### DISCUSSION THREAD

**Discussion by mariuz** [mariuz/flamerobin#144] [General]
New Flamerobin snapshot available! 0.9.3.5 with a few fixes and small features

https://github.com/mariuz/flamerobin/releases/tag/0.9.3.5

Take a look, test and comment!

**Comment by luronumen** [comment]
Could you add the possibility of hiding **System Packages** and **System Domains** just like already happen with **System Tables** and **System Roles**? (See: #146).

![SystemObjects](https://user-images.githubusercontent.com/8019466/103908065-d4eb0300-50e0-11eb-8ecc-e2161f6a0133.PNG)

**Reply by luronumen** [reply]
Two more suggestions:
- Rename **Generators** to **Sequences** (See: #54);
- Add **Indexes** (See: #48);

**Reply by arvanus** [reply]
Should this options be global or by database?
because I think that in some cases loading thousands of indexes in a remote database could turn the connection extremely slow

**Reply by Jdochoa** [reply]
Hi
I workin in  node for index, in two week's are ready. And i will work in a node for all system nodes.

**Reply by luronumen** [reply]
Hi @Jdochoa 

You can use the [RedExpert Firebird Admin](https://reddatabase.ru/downloads/redexpert/) as reference to implement *Indexes* node in FlameRobin:

![FlameRobin_Indexs](https://user-images.githubusercontent.com/8019466/104016268-c3fbc980-5194-11eb-8bde-4ac10e780984.PNG)

Thank you very much for your hard work to improve the FlameRobin!
